tests/kuttl/tests/keystone_scale/05-errors.yaml

```diff
@@ -0,0 +1,55 @@
+#
+# Check for:
+#
+# No KeystoneAPI CR
+# No Pods for KeystoneAPI CR
```

Comment:
also check for KeystoneAPI Route and Service gone

Reply:
Done